Automattic/jetpack
# Masterbar: Fix icon overlap issue at smaller resolutions

Related to https://github.com/Automattic/dotcom-forge/issues/8378

## Proposed changes:
Fix icon overlap issue at smaller resolutions matching [Calypso fix](https://github.com/Automattic/wp-calypso/pull/92967)
- Below 480px, I reduced the width of icons in masterbar down to 46px to make more space
- Below 480px I hid the "+" icon
- Below 360px I hide the reader icon

> [!IMPORTANT]  
> Some icons on the right panel could show in different places (related to https://github.com/Automattic/dotcom-forge/issues/8443)

### Desktop
| | |
| --- | --- |
| WP-Admin | ![image](https://github.com/user-attachments/assets/06d02b7c-1871-4e60-b81b-92053db20f8a) |
| Calypso | ![image](https://github.com/user-attachments/assets/4f14e35a-f0db-41ab-b830-3e2db6f5d5ef) |

### <780px
| | |
| --- | --- |
| WP-Admin | ![image](https://github.com/user-attachments/assets/c6c7cc8b-9047-4619-8895-908da91ae706) |
| Calypso | ![image](https://github.com/user-attachments/assets/6e31d951-a837-48da-befd-76799233bfff) |

### <480px
| | |
| --- | --- |
| WP-Admin | ![image](https://github.com/user-attachments/assets/10c0e3c5-efb6-4ecc-84ce-1ff5dc10f4a6) |
| Calypso | ![image](https://github.com/user-attachments/assets/5f734acb-68c4-46bf-8240-9917bfce3c6d) |

### <360px
| | |
| --- | --- |
| WP-Admin | ![image](https://github.com/user-attachments/assets/56ebcb83-aded-4eb4-9bb2-0aacef98acb6) |
| Calypso | ![image](https://github.com/user-attachments/assets/624c6d5a-865a-46e2-bb7a-99b3028a08fc) |

## Jetpack product discussion
- 

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Apply this PR using [the instructions in the first comment](https://github.com/Automattic/jetpack/pull/38551#issuecomment-2251367129).
* Go to `/wp-admin`
* Check the icon sizes and positions in the selected resolutions.
